src/test/java/com/systemmeltdown/robotlib/triggers/ToggleTriggerTest.java

```diff
@@ -0,0 +1,42 @@
+package com.systemmeltdown.robotlib.triggers;
+
+import edu.wpi.first.networktables.NetworkTableEntry;
+import edu.wpi.first.wpilibj.shuffleboard.BuiltInWidgets;
+import edu.wpi.first.wpilibj.shuffleboard.Shuffleboard;
+import edu.wpi.first.wpilibj.shuffleboard.ShuffleboardTab;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class ToggleTriggerTest {
+	@Test
+	public void testGetReturnsTrue() {
+    ShuffleboardTab tab = Shuffleboard.getTab("Test");
```

Comment:
Be more careful not to mix tabs and spaces; github shows tabs as six spaces so the formatting looks wrong. There should be a setting in the IDE to convert tabs to spaces when opening a file.